barretenberg/cpp/src/barretenberg/dsl/acir_format/ivc_recursion_constraint.test.cpp

```diff
@@ -75,39 +76,11 @@ class IvcRecursionConstraintTest : public ::testing::Test {
         };
     }
 
-    /**
```

Comment:
Removing this test because it was a pain to keep updating and is also no longer relevant because the kernel circuits will not do anything with public inputs anymore since the introduction of the databus.